generate_ide_files.bat

```diff
@@ -1,17 +1,26 @@
 
 cd RobotCode/snobot2018
```

Comment:
This is the magic script that takes the gradle build system and creates eclipse .project and .classpath files.  Sometimes updating it can get funky, so I added a clean step when you need to rebuild it.